docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-merge-pr.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: git-merge-pr
+description: Merges a specified open pull request.
+---
+
+# `git-merge-pr`
+
+`git-merge-pr` merges a specified open pull request. This step commonly follows a
+[`git-open-pr` step](git-open-pr.md) or can be used independently to merge existing
+pull requests.
```

Comment:
"or can be used independently to merge existing pull requests" may technically be true, but I think it feels like an invitation to all sorts of bizarre scenarios. Who or what would have opened that PR? We know that the ideal is that Kargo implements the entirety of one's promotion process. I just want our language to not open the door to things we don't recommend.

```suggestion
`git-merge-pr` merges an open pull request. This step commonly follows a
[`git-open-pr` step](git-open-pr.md).
```